Dockerfile

```diff
@@ -0,0 +1,24 @@
+# For more information, please refer to https://aka.ms/vscode-docker-python
+FROM python:3.8-slim-buster
+
+EXPOSE 8000
+
+# Keeps Python from generating .pyc files in the container
+ENV PYTHONDONTWRITEBYTECODE=1
+
+# Turns off buffering for easier container logging
+ENV PYTHONUNBUFFERED=1
+
+# Install pip requirements
+ADD requirements.txt .
+RUN python -m pip install -r requirements.txt
+
+WORKDIR /app
+ADD . /app
```

Comment:
No, this will add *everything* in this directory into the container, *including the UI source*. I had all the backend-related items in a directory separate from UI for a reason.

Reply:
I reorganized the projects back to their respective directories and this is no longer an issue.